langchain_tools_demo/main.py

```diff
@@ -66,13 +66,12 @@ async def index(request: Request):
     """Render the default template."""
     # Agent setup
     agent = await get_agent(request.session)
-    print(request.session["history"])
     return templates.TemplateResponse(
         "index.html",
         {
             "request": request,
             "messages": request.session["history"],
-            "client_id": request.session.get("client_id"),
+            "client_id": os.getenv("CLIENT_ID"),
```

Comment:
Can we move this to the file level? I don't think we want to do a syscall on every index call 
```suggestion
CLIENT_ID = os.getenv("CLIENT_ID")
# ...
            "client_id": CLIENT_ID,
```